line-bot-model/src/main/java/com/linecorp/bot/model/event/message/TextMessageContent.java

```diff
@@ -30,7 +31,6 @@
 @JsonTypeName("text")
 @Value
 @Builder(toBuilder = true)
-@AllArgsConstructor(onConstructor = @__(@Deprecated)) // TODO: Remove next release. Use builder() instead.
```

Comment:
Remove this constructor to care EventTestUtil.java...
I don't want to add a new constructor in this version...